### DISCUSSION TITLE
there was a problem with the installation

### DISCUSSION THREAD

**Discussion by er-fy** [open-mmlab/mmcv#2428] [General]
**Checklist**
![image](https://user-images.githubusercontent.com/76715891/202396008-1f459406-cdce-469b-bf9a-2ae902dfe158.png)

1. I have searched related issues but cannot get the expected help.
2. I have read the FAQ documentation but cannot get the expected help.

**Comment by zhouzaida** [comment]
Hi, could you provide more information like installing commands and pytorch versions?
